cicd/devnet/terraform/iam.tf

```diff
@@ -0,0 +1,28 @@
+# IAM policies
+data "aws_iam_policy_document" "xdc_ecs_tasks_execution_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
```

Comment:
why need assume role permission?

Reply:
I copy it from here
https://github.com/XinFinOrg/XDPoSChain/pull/259/files#diff-921bf923384362ec5f9d6448c80cdfb1383e8027738514422fe66d079f573dfdL87

Reply:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html
I think this means ecs task can assume this role